pkg/pillar/cmd/nim/nim.go

```diff
@@ -489,38 +492,19 @@ func (n *nim) initPublications() (err error) {
 		return err
 	}
 
-	n.pubWwanStatus, err = n.PubSub.NewPublication(
-		pubsub.PublicationOptions{
-			AgentName: agentName,
-			TopicType: types.WwanStatus{},
-		})
-	if err != nil {
-		return err
-	}
-
-	n.pubWwanMetrics, err = n.PubSub.NewPublication(
-		pubsub.PublicationOptions{
-			AgentName: agentName,
-			TopicType: types.WwanMetrics{},
-		})
-	if err != nil {
-		return err
-	}
-
-	n.pubWwanLocationInfo, err = n.PubSub.NewPublication(
+	n.pubCachedResolvedIPs, err = n.PubSub.NewPublication(
 		pubsub.PublicationOptions{
 			AgentName: agentName,
-			TopicType: types.WwanLocationInfo{},
```

Comment:
Maybe it's because of the diff, but do we have some kind of different information split now? (I don't see WwanLocationInfo topic)

Reply:
WwanLocationInfo, WwanMetrics and WwanStatus is now published by the wwan microservice, not NIM.
See: https://github.com/milan-zededa/eve/blob/modem-manager/pkg/wwan/README.md#inputoutput